syntax = "proto3";

package admin;

service AdminService {
  rpc GetAdmin (GetAdminRequest) returns (Admin);
  rpc CreateAdmin (CreateAdminRequest) returns (Admin);
  rpc GetAdmins (Empty) returns (AdminList);
}

message Admin {
  int32 id = 1;
}

message GetAdminRequest {
  int32 id = 1;
}

message CreateAdminRequest {
  int32 id = 1;
}

message AdminList {
  repeated Admin admins = 1;
}

message Empty {}